lua/entities/gmod_tardis/modules/cl_music.lua

```diff
@@ -42,21 +42,21 @@ function ENT:ResolveMusicURL(url)
 					local tbl=util.JSONToTable(body)
 					if tbl then
 						if tbl.success then
-							LocalPlayer():ChatPrint("Playing: "..tbl.title)
+                            TARDIS:Message(LocalPlayer(), "Playing: " ..tbl.title)
 							self:PlayMusic(api.."play?id="..id,true)
 						else
-							LocalPlayer():ChatPrint("ERROR: Failed to load ("..(tbl.err and tbl.err or "Unknown reason")..")")
+                            TARDIS:Message(LocalPlayer(), "ERROR: Failed to load ("..(tbl.err and tbl.err or "Unknown reason")..")")
```

Comment:
Same here and in all ERROR messages: TARDIS:ErrorMessage() would be better than TARDIS:Message()